apps/communications/contacts/test/marionette/export_sdcard_test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+var fs = require('fs');
+
+marionette('Export to SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
```

Comment:
Defining the prefs here is not needed, there already are under `Contacts.config`.